src/documentation/wiki/bootloader-client-format.md

```diff
@@ -1,53 +0,0 @@
----
```

Comment:
why was this removed? Shouldn't this go to cfclient firmware doc repo perhaps under the development header?

Reply:
aha misted that. no worries then!